styles/styles.css

```diff
@@ -0,0 +1,358 @@
+    text-decoration: none;
+}
+
+li a {
+    color: var(--lichtgrijs);
+    text-decoration: none;
+}
+li a:visited {
+    color: var(--lichtgrijs);
+}
+a:hover {
+    cursor: auto;
+}
+
+ul {
+    display: flex;
+    justify-content: space-between;
+
+    flex-basis: 400px;
+    padding: 20px;
+    list-style: none;
+}
+
+header {
+    background-color: var(--blauw);
+    background: linear-gradient(45deg, #4845E4, #5722CD);
+    /*background: linelinear-gradient(.25turn, --blauw, 10%, --paars);*/
+    color: var(--lichtpaars);
+}
+
+.header-content {
+    text-align: center;
+    padding: 20px;
+    margin: 20px;
+}
+
+.header-text {
+    padding: 20px;
+    /*margin: auto;*/
+}
+
+.btn_header {
+    border-radius: 60px;
+    padding: 20px 40px;
+    /*margin: 0 20px;*/
+    background-color: var(--roze);
+    color: var(--lichtpaars);
+    border: none;
+    box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+    text-transform: uppercase;
+}
+
+.header-image {
+    width: 40px;
+
+}
+
+
+/* //////// ----------- END OF HEADER ----------- //////// */
+
+/* //////// ----------- MAIN ----------- //////// */
+/* //////// ----------- SECTION 1 - HELLO ----------- //////// */
+
+.outer-container__main-section1 {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    flex-wrap: wrap;
+    margin-top: 50px;
+    width: 100%;
+}
+
+.inner-container__main-section1 {
+    flex-basis: 1000px;
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0 20px;
+}
+
+.section-hello__textbox {
+    color: var(--paars);
+    text-align: center;
+    /*width: 550px;*/
+    padding: 20px 220px;
+}
+
+.section-hello__textbox h3 {
+    margin: 20px;
+    font-weight: normal;
+}
+
+.inner-container p {
+    color: var(--lichtgrijs);
+    padding-bottom: 40px;
+}
+
+.section__link {
+    text-decoration: none;
+    color: var(--paars);
+}
+
+
+/* //////// ----------- SECTION 2 - WORK ----------- //////// */
+
+.outer-container__main-section2 {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    flex-wrap: wrap;
+    width: 100%;
+    margin-block: 125px;
+}
+
+.main-article-container {
+    display: flex;
+    flex-direction: row;
+    margin: 50px 100px;
+    /*padding: 100px;*/
+}
+
+.main-article-container:nth-child(even) {
+    display: flex;
+    flex-direction: row-reverse;
+}
+
+.article__textbox {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 50px;
+    margin: 0 50px 100px;
+}
+
+.article__textbox p {
+    margin-top: 30px;
+}
+
+.article__textbox-image {
+    width: 10%;
+    margin-top: 20px;
+}
+
+p a {
+    text-transform: uppercase;
+    color: var(--paars);
+    font-size: small;
+}
+
+
+.inner-container__image {
+    width: 100%;
+    box-shadow: rgba(0, 0, 0, 0.45) 0 25px 20px -20px;
+}
+
+.image-wrapper {
+    flex-basis: 1200px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    /*padding: 50px;*/
+    /*margin: 50px;*/
+}
```

Comment:
Vervolgens zet je hier
```suggestion
    display: flex;
    align-items: center;
}
```
Dan staat de afbeelding gecentreerd met de naastgelegen tekst

Reply:
![Scherm­afbeelding 2023-06-17 om 20 28 09](https://github.com/msnoodijk001/Websiteopdracht/assets/23332341/656876a4-066c-42d8-979f-b8bcfcbffe8c)